index.html

```diff
@@ -0,0 +1,119 @@
+                    <li><a href="">News</a></li>
+                    <li><a href="#">Sponsors</a></li>
+                    <li class="call"><a href="#">Call us</a></li>
+                </ul>
+            </div>
+        </header>
+        <div class="mobile-menu">
+            <i class="fas fa-times cancel"></i>
+            <ul class="menu-list">
+                <li><div class="box"><a href="index.html">Home</a></div></li>
+                <li><div class="box"><a href="about.html">About</a></div></li>
+                <li><div class="box"><a href="#">Blog</a></div></li>
+                <li><div class="box"><a href="#">News</a></div></li>
+
+            </ul>
+        </div> 
+        <div class="head">
+            <div class="header-headings">
+                <h3 class="header-one">An Online Education for All!</h3>
+                <h1 class="header-two">MASSIVE OPEN ONLINE COURSE SUMMIT</h1>
+            </div>
+            <div class="header-para">
+                <p>A Global online conference that discusses all possible approaches to adopt for the Online Education Program.</p>
+            </div>
+            <div class="header-cont">
+                <h2 class="para-time">2022.04.15(Thurs) ~ 16(Fri)</h2>
+                <p class="para-contact">@globalonlineprogram.com</p>
+            </div>   
+        </div>
+        <section class="main">
+            <h3 class="program">Main Program</h3>
+            <div class="big-forum">
+                <div class="forum">
+                    <i class="fas fa-laptop"></i>
+                    <h3>Lecture</h3>
+                    <p>listen to various speakers introducing what MOOC stands for</p>
+                </div>   
+                <div class="forum">
+                    <i class="fas fa-code"></i>
+                    <h3>Approaches</h3>
+                    <p>Challenges for the Online Education</p>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-atom"></i>
+                    <h3>Forum</h3>
+                    <P>Have time to share your thoughts and opinions with experts.</P>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-code"></i>
+                    <h3>Status</h3>
+                    <P>Different online Educational platforms</P>
+                </div>
+                <div class="forum">
+                    <i class="fas fa-database"></i>
+                    <h3>Workshop</h3>
+                    <P>Collection of all Questions and Answers</p>
+                </div>
+
+            </div>
+            <P><u>SEE THE WHOLE PROGRAM</u></P>
+            <button type="button" class="join">Join the MOOC Summit 2022</button>
+                                    
+        </section>
+        <section class="speakers">
+        </section>
+        <button type="button" class="more">MORE<img src="mooc-images/arrow.png" alt="the arrow to find more"></button>
```

Comment:
- [ ] OPTIONAL: After this button the rest of the content is not visible on the index page, please kindly add the footer.